mopidy_mpris/player.py

```diff
@@ -91,9 +91,7 @@ def PlayPause(self):
         state = self.core.playback.get_state().get()
         if state == PlaybackState.PLAYING:
             self.core.playback.pause().get()
-        elif state == PlaybackState.PAUSED:
-            self.core.playback.resume().get()
-        elif state == PlaybackState.STOPPED:
+        else:
             self.core.playback.play().get()
```

Comment:
Again, I don't remember how this works. Will `.play()` correctly resume from the middle of the track where you paused?

Reply:
It seems so, https://docs.mopidy.com/stable/_modules/mopidy/core/playback/#PlaybackController.play


```py
elif tl_track is None and self.get_state() == PlaybackState.PAUSED:
    self.resume()
    return
```